src/components/UI/Display/MAPI/workernodes/WorkerNodesNodePoolActions.tsx

```diff
@@ -69,7 +71,22 @@ const WorkerNodesNodePoolActions: React.FC<IWorkerNodesNodePoolActionsProps> = (
                         onBlurHandler();
                       }}
                     >
-                      Rename
+                      <Text>Rename</Text>
```

Comment:
Also wrapped the rename button in a `<Text />` component, since its font size was slightly different compared to the delete button